bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BKException.java

```diff
@@ -439,12 +439,22 @@ public BKLedgerIdOverflowException() {
      * Extract an exception code from an BKException, or use a default if it's another type.
      */
     public static int getExceptionCode(Throwable t, int defaultCode) {
-        if (t instanceof BKException) {
+        if (t == null) {
```

Comment:
can you explain why we are adding this? and please add a comment here

Reply:
Will add to javadoc